### ISSUE TITLE
[Word 2007] file cannot be opened in Word 2007

### ISSUE THREAD

**Issue by trevorHsu** [dolanmiu/docx#1113]
**Problem:**
The docx file cannot be opened  in Word 2007 with a tips said "Unspecified error". 

After testing several files, I found that the error location is  always on this code in /word/document.xml: 
<w:tblW w:type="auto" w:w="100%"/>

**What I expected:**
open the generated docx file in Word 2007

**Comment by anti-the-social** [comment]
Normally I'm using `<w:tblW w:type="pct" w:w="100%"/>`, but in order to test your problem I've disassembled .docx and changed "pct" to "auto" and it doesn't seem to be your problem since it opened just fine.

At the current state of docx js does not verify correctness of the file so there are lot of things you can do to have document that is not possible to open. In most cases it is connected to the structure of elements (paragraph into paragraph and such kind of things) The second popular reason for me is the numbering. (I don't remember exact cases now already, but I think it was anything like not correctly matched names between structure and names)

**Comment by trevorHsu** [comment]
Yeah, it seems pct is not suported in Word 2007, i use dxa instead, it works.

The image problem is not solved yet. 😢

**Comment by minhbvhp** [comment]
> Another problem, docx file with image is unable to open in Word 2007. [imageDoc.docx](https://github.com/dolanmiu/docx/files/7055683/imageDoc.docx)

I discovered that the error was caused by:
`export type DocPropertiesOptions = {
    readonly name: string;
    readonly description: string;
    readonly title: string;
};`
and when generate the ImageRun, the xml include tag:
`<wp:docPr id="1" name="ExampleImage" descr="" title=""/>`

Word 2007 can not render the xml with the attributes: "descr" and "title".
If you delete `descr="" title=""` , everything work.
I created a pull request to fix this error in [https://github.com/dolanmiu/docx/pull/3073](url).
For now, I’ve worked around it by creating my own npm package at [https://www.npmjs.com/package/@minhbvhp/docx ](url).

Hope this will help you !
